tests/integration/workflows/fixtures/oai_arxiv_record_with_plots.xml

```diff
@@ -24,7 +24,7 @@
             </author>
           </authors>
           <title>Spectra generated by a confined softcore Coulomb potential</title>
-          <categories>math-ph math.MP</categories>
+          <categories>math-ph math.MP astro-ph astro-ph.IM</categories>
```

Comment:
The thing is, in the real world we don't have the luxury of crafting the records, because we have to work with whatever the users throw at us, even if invalid or wrong. If we are in the mindset that we're allowed to deal with the *nice* case, we're not going to be prepared for the ugliness that is going to hit us later.